learning-bitmark/working-with-bitmarks/using-bitmark-shares.md

```diff
@@ -0,0 +1,631 @@
+# Bitmark shares
+
+In some cases, the ownership of a property is shared between different parties or people. To support those cases, the Bitmark Property System provides a feature called the **[bitmark shares]()**.
+
+Any owner of a Bitmark Certificate is able to:
+
+* [Create bitmark shares](#creating-bitmark-shares) from that certificate (or bitmark).
+
+* [Grant bitmark shares](#granting-bitmark-shares-to-another-account) to another account.
+
+* [Swap bitmark shares](#swapping-bitmark-shares) with others.
+
+
+Currently, only the Bitmark CLI supports performing those transactions of bitmark shares.
+
+> **NOTE:** Transactions related to the bitmark share require a transaction fee. On the bitmark network, the fee can be pay by BTC or LTC
+>
+> * Real BTC and/or LTC are used to pay for transaction fee on the Bitmark main chain
+>
+> * Testnet BTC and/or LTC are used to pay for transaction fee on the Bitmark testing chain
+
+<br>
+<br>
+
+## Bitmark shares related records
+
+There are three records related to the Bitmark shares are stored on the Bitmark blockchain
+
+* *Balance Record* - created by a bitmark owner to permanently set the total number of a particular share for the bitmark.
+
+* *Grant Record* - created by an owner to grant an amount of his share balance to another owner.
+
+* *Swap Record* - created by two owners to swap their shares of different bitmarks together.
+
+<br>
+<br>
+
+## Prerequisites of working with bitmark shares using the Bitmark CLI
+
+Refer the [Payment on Bitmark CLI](payment-on-bitmark-cli.md) document to:
+
+* Install the Bitmark-Wallet for transaction payments.
+
+<br>
+<br>
+
+## Creating bitmark shares
+
+Any bitmark owner is able to create shares of a particular bitmark, that action divides the bitmark into a number of shares
+
+* The initial number of shares is not changeable after created.
+
+* Once the bitmark is converted into shares, it cannot be transferred any more. The provenance of the bitmark is terminated.
+
+* All the created shares are alocated to the owner.
+
+* A share creation transaction costs 0.002 LTC or 0.0002 BTC as the transaction fee
+
+<br>
+
+### Creating bitmark shares from a bitmark
+
+  ```shell
+    $ bitmark-cli -n <network> -i <identity> \
+      share -t <txid> -q <quantity>
+  ```
+
+  >The `share` command is to create shares of a bitmark. The options:
+  >
+  >* `identity` - The bitmark owner's identity.
+  >
```

Comment:
I think it's always best to be as clear as possible in this sort of documentation. Rather than saying the "owner's identity", it'd be better to specify _precisely_ what the identifier is, as I might think it's the public key associated with the account or something else. (Here, it appears to be the name associated with the identity in the wallet.)

Reply:
It is the identity of the Bitmark Account configured in the Bitmark-CLI.

Reply:
Updated to:
>* `identity` - The bitmark owner's identity which is defined during creating a new Bitmark Account using the Bitmark CLI and stored in the CLI config file.